gnome-base/gnome-extra-apps/metadata.xml

```diff
@@ -7,8 +7,8 @@
 </maintainer>
 <use>
 	<flag name="games">Install Gnome Games</flag>
+	<flag name="localsearch">Install the <pkg>app-misc/tinysparql</pkg> indexer and the GNOME packages that require it</flag>
 	<flag name="share">Install the <pkg>gnome-extra/gnome-user-share</pkg> personal file sharing tool</flag>
 	<flag name="shotwell">Install the <pkg>media-gfx/shotwell</pkg> photo manager</flag>
-	<flag name="tracker">Install the <pkg>app-misc/tracker</pkg> indexer and the GNOME packages that require it</flag>
```

Comment:
this is purely educational and no instruction to fix this PR, but: 45.2 is still around, so you can't remove this.